pkgs/development/python-modules/dbus-next/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "dbus-next";
+  version = "0.1.4";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "dbus_next";
+    sha256 = "sha256-GxlCv/zIyaW9aDQlffInpV7ijgfdQT6tgt3SMRVlI2M=";
+  };
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
The warning can be ignored. It seems some file is missing.

Reply:
I've manage to enable and fix the testing - mostly - by following the Makefile. Two test input files that are in the source dir are missing in the build dir (`test/data/messages.json` and `test/data/introspection.xml`). What is the correct Nix way to make sure they are present?

Reply:
The PyPi tarball just didn't include the required files, but pulling for GitHub seems to fix it.